cluster-endpoints/src/grpc_subscription.rs

```diff
@@ -272,6 +203,63 @@ pub fn from_grpc_block_update(
     ProducedBlock::new(inner, commitment_config)
 }
 
+fn map_compute_budget_instructions(message: &VersionedMessage) -> (Option<u32>, Option<u64>) {
+    let cu_requested_cell: OnceCell<Option<u32>> = OnceCell::new();
```

Comment:
Onecell seems little useless with option, eigher should go with OnceCell or Option.